Gemfile

```diff
@@ -34,6 +32,8 @@ group :development, :test do
 end
 
 group :development do
+  # Use sqlite3 as the database for Active Record
+  gem 'sqlite3', '~> 1.4'
```

Comment:
question: Why not pg for development as well?

Reply:
That will work as well, but we will need to configure a couple more things and possibly migrate the db as well. Since the scope of the PR was to prepare the app for deployment onto Heroku, I suppose the instructor didn't go into that.

Question
1. When Heroku builds the app, does it not read from `database.yml`? I noticed that it still points to sqlite inside the production scope of that file.
2. Is it a better option to keep dev and prod database as the same type to mimic each other as close as possible?

Reply:
1. database.yml is what will be used to determine which db to use. Just cause you put in this gemfile doesn't mean it will work. You might need to do some slight tweak on database.yml to use postgres
2. Yes. that is the goal.